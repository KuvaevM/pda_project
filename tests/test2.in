start=[S]
[S] = `(`[S][R][S] | `(`[R][L];
[R] = `)`;