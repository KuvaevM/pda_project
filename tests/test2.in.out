States: q0 
Input alphabet: (, )
Stack alphabet: R, L, S
Start state: q0
Start stack element: S
Rules: {
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(()
			Stack symbol: NonTerminal(S)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(NonTerminal(S), NonTerminal(R), NonTerminal(S))
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal(()
			Stack symbol: NonTerminal(S)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(NonTerminal(R), NonTerminal(L))
		}
	}
	Rule: {
		Left: {
			State: q0
			Input symbol: Terminal())
			Stack symbol: NonTerminal(R)
		}
		Right: {
			State: q0
			Stack sequence: Sequence(Empty)
		}
	}
}
